services/dy-dash/cc-rabbit-0d/tests/requirements.txt

```diff
@@ -2,4 +2,4 @@ docker
 jsonschema
 pytest
 pyyaml
-pylint
\ No newline at end of file
+pylint~=2.4.4 # bug in 2.5.0 issue https://github.com/PyCQA/pylint/issues/3524 is resolved
```

Comment:
Please comment as ``FIXME: `` so we can trace it

Reply:
done